### ISSUE TITLE
Endpoint pod count is not expected, Timed out after 900s running get("", True, "noobaa-s3=noobaa")

### ISSUE THREAD

**Issue by tiffanyn108** [red-hat-storage/ocs-ci#8843]
Test case: tests.manage.mcg.test_endpoint_autoscale.TestEndpointAutoScale.test_scaling_under_load
Test Logs: 
https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/557/15916/750338/750398/750399/log

```
Message: ocs_ci.ocs.exceptions.TimeoutExpiredError: Timed out after 900s running get("", True, "noobaa-s3=noobaa")
Type: None

Text:
self = <tests.manage.mcg.test_endpoint_autoscale.TestEndpointAutoScale object at 0x7fcb331b6f40>
mcg_job_factory = <function mcg_job_factory.._factory at 0x7fcb21f1eb80>

```

**Comment by tiffanyn108** [comment]
Closing this issue since PR is merged.

**Comment by udaysk23** [comment]
Re-opening this issue since we are hitting this on latest build
https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#ocs/launches/557/17216/837741/837813/837814/log?logParams=history%3D837814%26page.page%3D1

**Comment by tiffanyn108** [comment]
@udaysk23 I don't see this test failing on 4.15 or 4.14.
Can you provide the recent failing log for this issue?

**Comment by tiffanyn108** [comment]
Test case failed in 4.14 which hasn't yet to backport.  Backport PR#8857 will fix the issue.

**Comment by tiffanyn108** [comment]
Closed this issue since #9508 is merged.
